modules/031-linstor/docs/CONFIGURATION.md

```diff
@@ -98,3 +98,222 @@ In case your application does not support high availability and runs in a single
+     snap.linstor.csi.linbit.com/type: S3
+     snap.linstor.csi.linbit.com/remote-name: backup-remote               # Linstor remote name.   
+     snap.linstor.csi.linbit.com/allow-incremental: "false"               # Whether to use incremental copies. 
+     snap.linstor.csi.linbit.com/s3-bucket: snapshot-bucket               # The name of the S3 bucket, for data storage.
+     snap.linstor.csi.linbit.com/s3-endpoint: s3.us-west-1.amazonaws.com  # S3 endpoint URL.
+     snap.linstor.csi.linbit.com/s3-signing-region: us-west-1             # The name of the S3 region. 
+     # Use virtual hosted–style or path-style S3 URL. 
+     # https://docs.aws.amazon.com/AmazonS3/latest/userguide/VirtualHosting.html
+     snap.linstor.csi.linbit.com/s3-use-path-style: "false"    
+     # Secret, containing the access key and secret key of access to S3 bucket.
+     csi.storage.k8s.io/snapshotter-secret-name: linstor-csi-s3-access
+     csi.storage.k8s.io/snapshotter-secret-namespace: storage
+   ---
+   kind: Secret
+   apiVersion: v1
+   metadata:
+     name: linstor-csi-s3-access
+     namespace: storage
+   immutable: true
+   type: linstor.csi.linbit.com/s3-credentials.v1
+   stringData:
+     access-key: *!ACCESS_KEY*  # S3 access key.
+     secret-key: *!SECRET_KEY*  # S3 secret key.
+   ```
+
+1. Get the snapshot id for recovery.
+
+   Get a list of snapshots, and select the one you need (the `SnapshotName` column):
+
+   ```shell
+   kubectl exec -n d8-linstor deploy/linstor-controller -- linstor backup list <backup-remote-name>
+   ```
+
+   , where the `<backup-remote-name>` is the remote name used in the `VolumeSnapshotClass`.
+
+1. Create a `VolumeSnapshotContent`, pointing to a specific snapshot id.
+
+   Example:
+
+   ```yaml
+   apiVersion: snapshot.storage.k8s.io/v1
+   kind: VolumeSnapshotContent
+   metadata:
+     name: restored-snap-content-from-s3
+   spec:
+     deletionPolicy: Delete
+     driver: linstor.csi.linbit.com
+     source:
+       snapshotHandle: *!snapshot_id*                        # Snapshot ID .  
+     volumeSnapshotClassName: linstor-csi-snapshot-class-s3  # The name of the VolumeSnapshotClass, with access to the S3 storage.
+     volumeSnapshotRef:
+       apiVersion: snapshot.storage.k8s.io/v1
+       kind: VolumeSnapshot
+       name: example-backup-from-s3                          # The name of the VolumeSnapshot (will be created below).
+       namespace: storage
+   ```
+
+1. Create a `VolumeSnapshot`, pointing to the created `VolumeSnapshotContent`.
+
+   Example:
+
+   ```yaml
+   apiVersion: snapshot.storage.k8s.io/v1
+   kind: VolumeSnapshot
+   metadata:
+     name: example-backup-from-s3
+     namespace: storage
+   spec:
+     source:
+       volumeSnapshotContentName: restored-snap-content-from-s3 # The name of the VolumeSnapshotContent created earlier.
+     volumeSnapshotClassName: linstor-csi-snapshot-class-s3     # The name of the VolumeSnapshotClass, with access to the S3 storage.
+   ```
```

Comment:
This is needed only for recovering into another cluster/namespace, isn't it?

Reply:
Made the changes. Please take a look on a [stage](https://deckhouse.stage.flant.com/en/documentation/v1/modules/031-linstor/configuration.html#restoring-from-a-backup) @kvaps